docs/docs/recipes.md

```diff
@@ -192,45 +187,34 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
 
 ## Constraining task placement / static partitioning
```

Comment:
We already have a separate doc for this: https://mesosphere.github.io/marathon/docs/constraints.html where we can point to. The recipe part of it is to give you some ideas how to use it.